contracts/aadex/Wasd3rDexAccountManager.sol

```diff
@@ -12,15 +12,13 @@ import './Wasd3rDexTokenManager.sol';
  */
 abstract contract Wasd3rDexAccountManager is Wasd3rDexTokenManager {
   /**
-   * @param isValid the deposit info is valid or invalid
    * @param amount the user deposited token amount
    * @param lastDepositBlockNo the block number when a user deposited the token lastly
    * @param lastDepositBlockNoIdx deposit index in the same block of the last deposit. If only once, it would be 0.
    * @param lastWithdrawtBlockNo the block number when a user withdrew the token lastly
    * @param lastWithdrawBlockNoIdx withdraw index in the same block of the last withdraw. If onle once, it would be 0.
    */
   struct DexDepositInfo {
-    bool isValid;
```

Comment:
Remove the validation of a token account.
Because the `isValid` flag in `DexAccountValid` is enough to enable/disable an account.